index.html

```diff
@@ -56,7 +56,8 @@
     </section>
 
     <section id="sotd">
-        <p>This specification is an update to [[PointerEvents]] which was shipped broadly only by Microsoft Internet Explorer and Microsoft Edge (though a further independent and mostly interoperable implementation was present in a pre-release build of Mozilla Firefox when the Pointer Events specification was published as a W3C Recommendation). Level 2 includes editorial clarifications, new features and minor breaking changes that address certain limitations and concerns that have been raised about aspects of the design, in an effort to enable wider browser adoption.</p>
+        <p>This specification is an update to [[PointerEvents2]] which was shipped broadly by Google Chrome and Microsoft Edge and Mozzila Firefox.
+           Level 3 includes editorial clarifications, new features to enable more use cases, in an effort to enable wider developer and browser adoption.</p>
```

Comment:
This change does not seem relevant to the new single-directional touch-action values. I think they should be moved to a different PR.